CODE_OF_CONDUCT.md

```diff
@@ -105,9 +97,7 @@ Violating these terms may lead to a permanent ban.
 
 ### 4. Permanent Ban
 
-**Community Impact**: Demonstrating a pattern of violation of community
-standards, including sustained inappropriate behavior,  harassment of an
-individual, or aggression toward or disparagement of classes of individuals.
+**Community Impact**: Demonstrating a pattern of violating community standards, such as persistent inappropriate behavior, harassment of individuals, or aggression toward or disparagement of groups of people. 
```

Comment:
I consider sustained is more precise in the goal of this point as it gives the meaning of a lesser frequency than persistent.